### PR TITLE
Remove routing decorators in `test_databases.py`

### PR DESCRIPTION
**Pull request by aminalaee**
Subset of #1481 drops usage of routing/middleware/lifestap decorators in tests/test_databases.py .

### CODE REVIEW

File: tests/test_database.py [hunk 1]
```diff
@@ -82,15 +77,27 @@ async def read_note(request):
...
+    routes=[
+        Route("/notes", endpoint=list_notes, methods=["GET"]),
+        Route("/notes", endpoint=add_note, methods=["POST"]),
+        Route("/notes/bulk_create", endpoint=bulk_create_notes, methods=["POST"]),
+        Route("/notes/{note_id:int}", endpoint=read_note, methods=["GET"]),
+        Route("/notes/{note_id:int}/text", endpoint=read_note_text, methods=["GET"]),
+    ],
+    on_startup=[startup],
+    on_shutdown=[shutdown],
+)
```
Comment (tomchristie) [inline tests/test_database.py:98]: 😌 - Yes.

**Review by lovelydinosaur — APPROVED** [review]

File: tests/test_database.py (hunk 1 above)
Comment (lovelydinosaur) [inline tests/test_database.py:98]: 😌 - Yes.

Comment (lovelydinosaur) [reply tests/test_database.py:98]: This is what I like to see.

Comment (lovelydinosaur) [reply tests/test_database.py:98]: So direct, so upfront, so relaxing.

Comment (aminalaee) [reply tests/test_database.py:98]: Exactly, you can view all routes/middlewares in one view. Much more readable.

**Pull request merged**